server/views/components/prisoner-split-view.njk

```diff
@@ -0,0 +1,24 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "./prisoner-summary-detail.njk" import prisonerSummaryDetail %}
+
+{% macro prisonerSplitView(prisonerEscortRecord, prisonRecord, personDetailsHeaderText='Details from Person Escort Record') %}
```

Comment:
Seemed like a good candidate for it's own component since the majority of the code is the same. The only one I didn't move over was the unexpected single match since it used prisonerSearchDetail() instead of prisonerSummaryDetail()

Reply:
Would be tempted to not default the heading and make it a mandatory argument - when its only 2 occurrences vs 1 there's not really a clear default.